ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -3158,3 +3158,23 @@ literals:
 - low_discharge_air_temperature_alarm:
   - ACTIVE
   - INACTIVE
+
+- soundpressurelevel_sensor:
+    fixed_min: -60
+    flexible_max: 200
+
+- zone_occupant_counter:
+    fixed_min: 0
+    flexible_max: 1000
+
+- zone_air_voc_aqi_sensor:
+    fixed_min: 0
+    flexible_max: 500
+
+- zone_air_smoke_aqi_sensor:
+    fixed_min: 0
+    flexible_max: 300
+
+- tamper_status:
```

Comment:
the sensor also has a boolean state which represents if the device has been moved or "tampered" with.

Reply:
Is this a critical field to have? Is it important for the end-users to know if the sensor has been moved?

Reply:
This information is provided to facility managers, so I would say this information is good for them to know, but not essentially critical.